charts/digger-backend/templates/backend-deployment.yaml

```diff
@@ -35,6 +35,11 @@ spec:
             secretKeyRef:
               name: {{ .Values.postgres.secret.existingSecretName }}
               key: postgres-password
+        {{- else if .Values.digger.postgres.existingSecretName }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.digger.postgres.existingSecretName }}
+              key: {{ .Values.digger.postgres.existingSecretKey }}
         {{- else }}
           valueFrom:
             secretKeyRef:
```

Comment:
I can't leave a comment outside of the diff, but the succeeding if block (lines 49-54) needs to also be removed. It's what breaks the template.

Reply:
https://github.com/diggerhq/helm-charts/pull/15/files#diff-204d7782307a1567576db307ef282e33ef9a56ab6024b064960f777651b15acaR32-R54

The "else" and second if condition with negation of the preceding if conditions result in two valuesFrom blocks when `and .Values.digger.postgres.existingSecretName (not .Values.postgres.enabled)`

Reply:
Yeah, sorry. I had already explained on Slack (and showed the diff on my fork that this PR is based off of) so I figured I didn't need to reiterate it.

Reply:
Making sense, removed!